contractcourt/briefcase.go

```diff
@@ -82,17 +81,6 @@ type ArbitratorLog interface {
 	// contract resolutions from persistent storage.
 	FetchContractResolutions() (*ContractResolutions, error)
 
-	// LogChainActions stores a set of chain actions which are derived from
-	// our set of active contracts, and the on-chain state. We'll write
-	// this et of cations when: we decide to go on-chain to resolve a
-	// contract, or we detect that the remote party has gone on-chain.
-	LogChainActions(ChainActionMap) error
-
-	// FetchChainActions attempts to fetch the set of previously stored
-	// chain actions. We'll use this upon restart to properly advance our
-	// state machine forward.
-	FetchChainActions() (ChainActionMap, error)
```

Comment:
I would remove this in the same commit where we stop using them, to prevent having dead code in master.